lms/djangoapps/course_api/api.py

```diff
@@ -0,0 +1,83 @@
+"""
```

Comment:
I feel like the name of this API is too generic.  Which aspect(s) of the course concept does it cover?  Is this the place where all access to information about courses will be contained?  I can see we are already aggregating information found in the course module (ie, metadata) as well as the much-debated "about" information.  I just want to ensure we set the proper boundaries for what information is 'in' this API, versus what information is 'out'.  

In the very the least, we should pluralize the app name to 'courses_api'.